src/api/routers/usersRouter.ts

```diff
@@ -54,8 +57,15 @@ export function createUsersRouter() {
   });
 
   usersRouter.get('/current/participant', async (req: UserRequest, res) => {
-    const participant = await req.user!.$relatedQuery('participant').withGraphFetched('types');
-    return res.status(200).json(participant);
+    const currentParticipant = await Participant.query().findOne({ id: req.user!.participantId });
+    const sites = await getSiteList();
+    const currentSite = sites.find((x) => x.id === currentParticipant?.siteId);
```

Comment:
nit: maybe we need a get site by id endpoint in uid2-admin later

Reply:
yep I agree!